README.md

```diff
@@ -105,24 +105,20 @@ To deploy the Infrastructure-as-Code (IaC) resources needed for this solution, p
     - Triggers a pop-up dialog box: 'Sign in with Google' prompting you to authenticate the Google Auth Library. Follow the directions to authenticate with your user account, which will then configure Application Default Credentials (ADC) using the impersonated service account credentials to be used by Terraform.
 
 1.  Create a terraform.tfvars file with the following variables:
-
-    ```sh
-    project_id = # Your Google Cloud project ID.
-    region = # The desired region for deploying single-region resources (e.g., "us-central1", "europe-west1").
-    vertex_ai_data_store_region = # The multiregion for your Agent Builder Data Store, the possible values are ("global", "us", or "eu").
-    docai_location = # Sets the location for Document AI service.
-    webui_domains = # Your domain name for Web UI access (e.g., ["webui.example.com"])
-    iap_access_domains = # List of domains granted for IAP access to the Web UI (e.g., ["domain:example.com"])
-    ```
-
-    - (Optional) By default, the Terraform script creates a new VPC network in the same project as other resources. You can use an existing VPC network instead by configuring the following optional terraform variables.
-
-      If using an existing VPC, you should validate that your existing vpc has firewall policies and DNS zones that enable the traffic pathways defined in [vpc.tf](enterprise-knowledge-solution/components/common-infra/terraform/vpc.tf), and grant Compute Network User on your shared VPC to the deployer service account.
-
-      ```sh
-      create_vpc_network = false # default is true
-      vpc_name = # The name of your existing vpc, (e.g., "myvpc")
-      ```
+      | Terraform variables         | Description                                                                                          |
```

Comment:
In my previous commit I changed this section back the to the shell code format, and it looks like you reverted it back again to the table. Why the table?

My view is that the lines of shell are easier to copy paste with a few clicks to modify the values. The table format is harder to use because they have to copy paste each line individual or reformat it somewhere else before they can put it in a shell.

Reply:
The users should clearly understand these setting and so I think we should keep this in a table form so that when they do it one at a time, they think rather than blank copy-paste.